client/state/plans/selectors.js

```diff
@@ -43,7 +48,7 @@ export const getPlan = createSelector(
  */
 export function getPlanRawPrice( state, productId, isMonthly = false ) {
 	const plan = getPlan( state, productId );
-	if ( ! plan || ! ( plan.raw_price || plan.raw_price === 0 ) ) {
+	if ( get( plan, 'raw_price', -1 ) < 0 ) {
```

Comment:
Yeah I think an existential operator would read better, but it basically boils down to fetch plan.raw_price if possible, otherwise default to -1. 

https://lodash.com/docs#get